### ISSUE TITLE
Will get object data when the trans key is incomplete

### ISSUE THREAD

**Issue by bigBear713** [bigBear713/nb-trans#27]
**version**: `v15.1.0`

When the key value is incomplete, the translated result is an object data, not a string data. Like this:
In translations.json
```json
{
  "level1":{
      "level2":{
          "level3":"hello world"
      }
  }
}
```
In template:
```html
<div>{{'level1.level2'|nbTrans}}</div>
```

Then the content in page is 
```html
<div>[object Object]</div>
```

Because the trans key is `level1.level2` which is incomplete, so the result is object data: `{ "level3":"hello world" }`
The complete key is `level1.level2.level3`，and the expected result is `hello world`

**Comment by bigBear713** [comment]
In the case, should try to get translated string from default lang.
If the result is always not a string, return empty result.

**Comment by bigBear713** [comment]
The translated sentence is valid only when it is of string type

**Comment by bigBear713** [comment]
> The translated sentence is valid only when it is a string type

Although `boolean` and `number` can be implicitly converted to `string` types, it would be more expected and better when let the developer provide the value of the string type directly

**Comment by bigBear713** [comment]
The bug has been fixed in `v16.0.0`
